std/range/package.d

```diff
@@ -7516,7 +7516,7 @@ public:
     /// Forward range primitives. Always present.
     @property auto front()
     {
-        assert(!empty, "Attempting to access front on an empty range");
+        assert(!empty, "Attempting to access front on an empty slides");
```

Comment:
@andralex Typically range construction functions are verbs, e.g. `map`, `filter`, `padLeft` so maybe `slide` would be better?